diff -rupN yacreader-10.0.0.old/YACReaderLibraryServer/CMakeLists.txt yacreader-10.0.0/YACReaderLibraryServer/CMakeLists.txt
--- yacreader-10.0.0.old/YACReaderLibraryServer/CMakeLists.txt	2026-05-01 09:43:00.000000000 +0200
+++ yacreader-10.0.0/YACReaderLibraryServer/CMakeLists.txt	2026-05-01 13:02:53.645770945 +0200
@@ -101,5 +101,5 @@ if(UNIX AND NOT APPLE)
 
     # systemd service
     install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/yacreaderlibraryserver.service"
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/systemd/user)
+        DESTINATION /usr/lib/systemd/user)
 endif()
